Problem with MySQL SSL connection

Hi Releem Team

I recently have activated SSL for my MariaDB clients. I´ve succesfuly connect all my db users except Reléemelos agent

Removed the server from the web Dashboard and try to reinstal the agent.

Here are the results:


% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 21341  100 21341    0     0   121k      0 --:--:-- --:--:-- --:--:--  121k

 * Installing dependences...


 * Downloading Releem Agent, architecture x86_64...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 21673  100 21673    0     0   169k      0 --:--:-- --:--:-- --:--:--  169k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 19.9M  100 19.9M    0     0  21.5M      0 --:--:-- --:--:-- --:--:-- 21.4M

 * Configure the application...

 * Detected service name for appling config

 * Configure catalog for copy recommend config

 * The /etc/mysql/my.cnf file is used for automatic Releem settings. 

 * Adding directive includedir to the MySQL configuration /etc/mysql/my.cnf.

 * Configure MySQL user for collect data

 * Using MySQL root user

 Connect to MySQL - successful 

 Created new user `releem`

 Connecting to MySQL with user `releem` - successfull 

 * Configure mysql memory limit

 * Saving variables to Releem Agent configuration

 - Adding API key to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding Releem Configuration Directory /opt/releem/conf to Releem Agent configuration: /opt/releem/releem.conf
 - Adding user and password mysql to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding Memory Limit to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding MySQL restart command to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding MySQL include directory to the Releem Agent configuration /opt/releem/releem.conf.
 - Adding autodetected hostname to the Releem Agent configuration: /opt/releem/releem.conf

 * Configure crontab...

 * Executing Releem Agent for first time...
INFO     : [Main] - 2024/05/28 22:28:22 main.go:85: Starting releem-agent of version is 1.16.0
INFO     : [Main] - 2024/05/28 22:28:22 config.go:46: Loading config /opt/releem/releem.conf
INFO     : [Main] - 2024/05/28 22:28:22 main.go:155: InstanceType is Local
ERROR    : [Main] - 2024/05/28 22:28:22 main.go:179:  GOT ERROR Connection failed Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
INFO     : [Worker] - 2024/05/28 22:28:22 runner.go:73:  * Collecting metrics to recommend a config...
ERROR    : [DbConf] - 2024/05/28 22:28:22 dbConf.go:39: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [DbInfo] - 2024/05/28 22:28:22 dbInfo.go:44: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [DbMetricsBase] - 2024/05/28 22:28:22 dbMetricsBase.go:43: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [Worker] - 2024/05/28 22:28:22 runner.go:254: Problem getting metrics from gatherer
INFO     : [Main] - 2024/05/28 22:28:22 main.go:85: Starting releem-agent of version is 1.16.0
INFO     : [Main] - 2024/05/28 22:28:22 config.go:46: Loading config /opt/releem/releem.conf
INFO     : [Main] - 2024/05/28 22:28:22 main.go:155: InstanceType is Local
ERROR    : [Main] - 2024/05/28 22:28:22 main.go:179:  GOT ERROR Connection failed Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [DbConf] - 2024/05/28 22:28:23 dbConf.go:39: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [DbInfo] - 2024/05/28 22:28:23 dbInfo.go:44: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [DbMetricsBase] - 2024/05/28 22:28:23 dbMetricsBase.go:43: Error 1045 (28000): Access denied for user 'releem'@'127.0.0.1' (using password: YES)
ERROR    : [Worker] - 2024/05/28 22:28:23 runner.go:254: Problem getting metrics from gatherer
INFO     : [Worker] - 2024/05/28 22:28:25 runner.go:51: Exiting

 * Installing and starting Releem Agent service to collect metrics..

 Installing Releem Agent - successful

 Restarting Releem Agent - successful

 * Enabling Performance schema and SlowLog to collect metrics...


 * Performance schema and SlowLog are enabled for metrics collection.

 * Releem Agent is successfully installed.

 * To view Releem recommendations and MySQL metrics, visit https://app.releem.com/dashboard


Thanks in advance for the answers

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,21 @@ Releem releases
 ---
 Information about releases of the Releem.
 
+Releem 1.17.0, 2024-06-30
+- Improved Weekly Reports
+- Added Alerts on CPU Utilization and Disk space. Closes #147
+- Added applying configuration without restart to Releem Agent.
+- Added SSL support to Releem Agent. Closes #310
+- Added collection information from performance_schema.file_summary_by_instance
+- Fixed Automatic installation doens't work in some cases. Closes #166
+- Fixed buffer recommendations more that total RAM. Closes #262
+
+Releem 1.16.0, 2024-05-31 ([What's New At Releem | May 2024](https://releem.com/blog/whats-new-at-releem-may-2024))
+- Added Security Checks
+- Added new event for partially applied configuration.
+- Added support of AWS RDS Aurora. Closes #308
+- Fixed Many events on MySQL restart. Closes #305
+
 Releem 1.15.0, 2024-04-30
 - Added new option to disable Query Cache manually. Feature request #289
 - Added Max Query Length option allows Releem to save full queries for analysis. Closes #291

